src/models/constants.json

```diff
@@ -39,6 +35,10 @@
     "party",
     "wedding",
     "culture",
+    "kids",
+    "movie",
```

Comment:
Call this 'film' or 'movies'

Reply:
yeah decided to go with `movie` instead of `film` cause `film` is more of a British way of saying it, and the society here is more american influenced :sweat_smile: , yeah but also i think being i'm picky here but `movie` makes more sense cause it's 1 event everytime and it's **a** category.